lldb/source/Plugins/SymbolFile/DWARF/DWARFDebugInfoEntry.cpp

```diff
@@ -309,10 +319,10 @@ void DWARFDebugInfoEntry::GetAttributes(DWARFUnit *cu,
     switch (attr) {
     case DW_AT_sibling:
     case DW_AT_declaration:
-      if (curr_depth > 0) {
+      if (seen.size() > 1 && !is_first_die) {
```

Comment:
```suggestion
      if (!is_first_die) {
```

I don't think the other condition is necessary.

Reply:
Good point!